app/components/dashboard/run/left-panel/template.hbs

```diff
@@ -41,13 +28,22 @@
                             <i class="fas fa-ellipsis-v clickable"></i>
                         </a>
                     </div>
+                    
                     {{#if displayTaleInstanceMenu}}
                         {{#click-outside action=(action (mut displayTaleInstanceMenu) false)}}
                             <div class="tale-instance-menu">
                                 <div class="ui vertical left menu transition">
                                     <a class="item" href="https://wholetale.readthedocs.io/en/stable/users_guide/run.html" target="_blank">
                                         Read the docs <div class="ui label transparent"><i class="fas fa-book"></i></div>
                                     </a>
+                                    <div class="ui left pointing dropdown link item">
+                                      Export
+                                      <div class="menu">
+                                        <div class="item" style="width: 200px; z-index:999;" {{action "exportBag" model.taleId}}>BagIt</div>
```

Comment:
I believe you can pass multiple arguments to an action like this: `{{action "exportTale" model.taleId "native"}}`

Also, can the width of this menu item match the others?